AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -332,22 +306,24 @@ - (MSIdentityConfig *)deserializeData:(NSData *)data {
 }
 
 - (BOOL)clearAuthData {
+  BOOL result = YES;
+  if (![self removeAccount]) {
+    MSLogWarning([MSIdentity logTag], @"Couldn't remove account data.");
+    result = NO;
```

Comment:
I would suggest using two flags here and `return accountSuccessfullyDeleted && authTokenSuccessfullyCleared;` at the end. But that's up to you - just would look more clear and we could assign the reult of the call directly to the flag.

Reply:
I'm not sure about using 2 flags. Because in that case these flags will be used only once in the return statement and won't be used separately. For me it looks not so much different from the current implementation. I would keep it as is. Thank you.